src/Video/BridgeOpenCV/ConvertAnITKGrayScaleImageToCVMat/Code.cxx

```diff
@@ -21,12 +21,6 @@
 #include "itkOpenCVImageBridge.h"
 
 // includes from OpenCV
-#include "cv.h"
-#if CV_VERSION_MAJOR > 2
-#  include "opencv2/opencv.hpp" // cv::imwrite
-#endif
-
-
 #if CV_VERSION_MAJOR > 2
```

Comment:
I think we don't need this conditional statement any more. At least not in an example. Just plain old `#include "opencv2/opencv.hpp"`.